operations/system_admin_toolkit/usage/SAT_and_IUF.md

```diff
@@ -89,3 +89,256 @@ INFO: Validating given input file example-bootprep-input-file.yaml
 INFO: Input file successfully validated against schema
 INFO: Skipping creation of CFS configurations based on value of --limit option.
 ```
+
+## Using `sat bootprep` with IUF generated input files
```

Comment:
I think this is great information to have in case it's needed, but I think it needs to be updated a bit to be less specific to the use case of creating management node CFS configurations and building management node images with an updated CSM version.

Reply:
Ryan fixed in this in his commit.